client/src/styles/Header/HeaderContent.css

```diff
@@ -0,0 +1,57 @@
+@import url("https://fonts.googleapis.com/css2?family=Lora&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+
+.header {
+  display: flex;
+  flex-direction: row;
+  margin-left: 50px;
+  margin-bottom: 85px;
+}
+
+.header-img {
+  margin-top: 5%;
+  margin-left: 80px;
+  width: 55%;
+  background-color: transparent;
+  margin-right: 5px;
+}
+.header-left {
+  width: 45%;
+  background-color: transparent;
+}
+img {
+  width: 90%;
+}
+h1,
+h5 {
+  font-family: "Lora", serif;
+  margin-top: 10%;
+  font-size: 50px;
+  word-spacing: 5px;
+  letter-spacing: 2px;
+  color: #003366;
+}
+h5 {
+  font-size: 20px;
+  letter-spacing: 0px;
+  word-spacing: 0px;
+}
```

Comment:
we can use **p** html tag instead of the heading tags, as we are altering the font-size of same.